common/src/main/java/com/wynntils/models/lootrun/LootrunModel.java

```diff
@@ -135,39 +175,78 @@ public void onChatMessage(ChatMessageReceivedEvent event) {
 
     @SubscribeEvent
     public void onWorldStateChanged(WorldStateEvent event) {
+        // The world state event is sometimes late compared to lootrun events (beacons, scoreboard)
```

Comment:
Is it randomly late, or always? Maybe we need to update the world event...

Reply:
randomly, around 1 in 5 times